hexrd/config/findorientations.py

```diff
@@ -231,12 +270,47 @@ def eta_step(self):
 
     @property
     def file(self):
-        temp = self._cfg.get('find_orientations:orientation_maps:file',
-                             default=None)
+        """Path of eta-omega maps file"""
+        #
+        # Get file name.  Because users often set file to "null", which
+        # returns a valid value of None, we have to check twice before setting
+        # it to the default value.
+        #
+        root = self.parent
+        if root.new_file_placement:
+            actmat = root.find_orientations._active_material_str()
+            dflt = f"eta-ome-maps-{actmat}.npz"
+            mapf = root.analysis_dir / dflt
+        else:
+            fname = '_'.join([root.analysis_id, "eta-ome_maps.npz"])
+            mapf = root.working_dir / fname
+
+        # Now check the YAML.
+        temp = self._cfg.get(
+            'find_orientations:orientation_maps:file',
+            default=None
+        )
         if temp is not None:
-            if not os.path.isabs(temp):
-                temp = os.path.join(self._cfg.working_dir, temp)
-        return temp
+            ptemp = Path(temp)
+            if ptemp.is_absolute():
+                mapf = ptemp
+            else:
+                mapf = root.working_dir / ptemp
+
+        return mapf
+
+    @property
+    def scored_orientations_file(self):
+        root = self.parent
+        if root.new_file_placement:
+            adir = root.analysis_dir
+            actmat = root.find_orientations._active_material_str()
+            sof = Path(adir) / f'scored-orientations-{actmat}.npz'
```

Comment:
```suggestion
            sof = adir / f'scored-orientations-{actmat}.npz'
```

The analysis dir is guaranteed to be a `Path` object now, right?

Reply:
Yes, the `analysis_dir` and the `working_dir` are both guaranteed to be a Path, regardless of whether new or old file placement is being used.